### PR TITLE
Add emacs style ctrl-a ctrl-e navigation

### PR DESCRIPTION
**Pull request by ChillerDragon**
Im sure im not the only one who keeps pressing ctrl-a and ctrl-e in the teewoods console and nothing happens .-.

### CODE REVIEW

File: src/game/client/components/console.cpp [hunk 1]
```diff
@@ -109,38 +109,48 @@ void CGameConsole::CInstance::OnInput(IInput::CEvent Event)
 {
 	bool Handled = false;
 
-	if(m_pGameConsole->Input()->KeyIsPressed(KEY_LCTRL) && m_pGameConsole->Input()->KeyPress(KEY_V))
```
Comment (def-) [inline src/game/client/components/console.cpp]: Not sure I like the switch to nested ifs, I liked being able to grep for KEY_CTRL and KEY_V and find where it is. Other opinions?

Comment (ChillerDragon) [reply src/game/client/components/console.cpp]: Agree. I decided to do so to increase performance by avoiding duplicated checks for the same key and some logical grouping of compose keys.

**Comment by def-** [comment]
Where else do we support emacs style inputs? I think home/end keys are good enough and common.

**Comment by ChillerDragon** [comment]
@def- as far as i know nowhere. But imo in the console it would make sense. Was thinking about ctrl-U as well.

**Comment by def-** [comment]
I don't think emacs users are our main target group. Word-style navigation sounds more like something people would be familiar with.

**Comment by ChillerDragon** [comment]
@def- is the teeworlds console more like a unix shell or more like word? :p
Im fine with word navigation in chat etc.

**Comment by def-** [comment]
Then at least remove the nested ifs again.

**Pull request merged**